src/chrome/content/rules/Madison-Logic.xml

```diff
@@ -49,14 +44,11 @@
 	*** Secured by us, doesn't trip MCB anyway
 
 -->
-<ruleset name="Madison Logic (partial)">
+<ruleset name="Madison Logic (partial)" default_off="failing tests">
```

Comment:
Some targets including `clk`, `jsc` and `st` work over HTTPS. It would be great if you can keep their rules. Please also try to document the non-functional host in the top comment, as suggested in [CONTRIBUTING.md#non-working-hosts](https://github.com/EFForg/https-everywhere/blob/master/CONTRIBUTING.md#non-working-hosts)

Reply:
Re-enabled and removed only non-functioning sites.

Is that the preferred style at https://github.com/EFForg/https-everywhere/blob/master/CONTRIBUTING.md#non-working-hosts - I've looked at an awful lot of the rules and not seen that style.

I can see it appears on ~1000 rules.  If that is the new style, I'll be happy to update my PRs accordingly.